P/PicoSAT/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PicoSAT"
+version = v"965.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://fmv.jku.at/picosat/picosat-965.tar.gz", "15169b4f28ba8f628f353f6f75a100845cdef4a2244f101a02b6e5a26e46a754")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/picosat-965
+if [[ ${target} == *musl* ]]; then
+    sed -i 's!sys/unistd.h!unistd.h!g' picosat.c
+fi
+if [[ ${target} == *mingw* ]]; then
+    sed -i 's!case X"$CC" in!case X"$target" in!g' configure.sh
+fi
+if [[ "${nbits}" == 32 ]]; then
+    ./configure.sh --shared --32
+else
+    ./configure.sh --shared
+fi
+
+make
+mkdir -p ${bindir} && cp ${WORKSPACE}/srcdir/picosat-965/{picosat${exeext},picomus${exeext},picomcs${exeext},picogcnf${exeext}} ${bindir}/
+mkdir -p ${libdir} && cp ${WORKSPACE}/srcdir/picosat-965/libpicosat.${dlext} ${libdir}/
```

Comment:
```suggestion
mkdir -p ${bindir} && cp {picosat,picomus,picomcs,picogcnf}${exeext} ${bindir}/.
mkdir -p ${libdir} && cp libpicosat.${dlext} ${libdir}/
```
much simpler, no? :slightly_smiling_face: